CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v1.0.0-beta1 (20 Feb 2017)
```

Comment:
What do you think about the second part of my comment? master tracking until 1.0.0